locations/spiders/gwk_travelex.py

```diff
@@ -4,7 +4,7 @@
 
 
 class TravelexSpider(scrapy.Spider):
-    name = "travelex"
+    name = "travelex_nl"
```

Comment:
The spider name should probably be `gwk_travelex_nl` ? The file name and class name should also match.